src/Simulation/QCTraceSimulator.Tests/Tests.Microsoft.Quantum.Simulation.QCTraceSimulatorRuntime.csproj

```diff
@@ -13,6 +13,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\Xunit\Microsoft.Quantum.Xunit.csproj" />
     <ProjectReference Include="..\CsharpGeneration\Microsoft.Quantum.CsharpGeneration.fsproj" IsQscReference="true" />
+    <ProjectReference Include="..\Simulators\Microsoft.Quantum.Simulators.csproj" />
```

Comment:
I don't have a good overview just looking at this PR how the project/package structure now works: Wasn't the reference before the other way round (i.e. the simulators project referencing the QCTraceSimulatorRuntime)? Why has that changed? How does that work if we were to ever ship each simulator separately?

Reply:
I'll need to double check this one; I may have added this speculatively when it isn't strictly necessary.

Reply:
Ah, I remember now why this was needed: when I split out the three simulators I had to remove the simulator reference from src\Simulators\Common\Simulators.Dev.props and add it explicitly to any projects that needed to refer to the QsharpCore simulator. So it's not a new dependency, just now it's direct instead of being via a props file.